news/12728.feature.rst

```diff
@@ -0,0 +1,5 @@
+``wheel`` is no longer explicitly listed as a build depepndency of ``pip``.
+``setuptools`` already injects this dependency in the ``get_requires_for_build_wheel()`` hook.
+This makes no difference for users of ``pip``.
+This makes no difference when building wheels of ``pip``.
+This avoids an unnecessary dependency on ``wheel`` when building the source distribution of ``pip``.
```

Comment:
Since this is not a user facing feature, I'd rename this news/12728.process.rst which is the best category we have now for this kind of change to the build process

Reply:
One could argue that it's a bugfix. But a "process"? This is a rather weird workaround.